src/theme/semanticTokens.ts

```diff
@@ -61,15 +61,15 @@ export const semanticTokens = {
       },
       input: {
         base: {
-          default: 'gray.50',
+          default: 'gray.100',
```

Comment:
Adds 50px to `background.input.base` for light them to give a bit more contrast to things, `gray.50` on white was pretty much invisible, making filled variant inputs weird to look at.

Similarly, 50px increments for others.

Reply:
Gomes 👨‍🎨